src/lib/Components/Picker.tsx

```diff
@@ -111,14 +111,18 @@ export class Picker extends React.Component<Props, State> {
   renderPicker = () => {
     const { options } = this.props
     const { isOpen, pendingSelected } = this.state
+    console.log("hii")
     return (
       <Portal>
         <Spring
           native
-          from={{ bottom: -260, progress: 0, opacity: 0 }}
-          to={isOpen ? { bottom: 0, progress: 1, opacity: 0.4 } : { bottom: -260, progress: 0, opacity: 0 }}
+          from={{ bottom: -280, progress: 0, opacity: 0 }}
+          to={isOpen ? { bottom: 0, progress: 1, opacity: 0.4 } : { bottom: -280, progress: 0, opacity: 0 }}
           onRest={this.handleSpringRest}
-          config={config.stiff}
+          config={{
+            ...config.stiff,
+            mass: 0.5,
```

Comment:
It looks like they changed the physics across a few versions, and things were a bit too springy. This makes it match how it was, but decreasing the mass of the menu.